src/platform/site-wide/sass/shame.scss

```diff
@@ -589,3 +589,10 @@ img[data-srcset] {
     width: 100%;
   }
 }
+
+.field-spokes-link-arrow {
+  width: 11px;
+  margin-left: 5px;
+  transition: all 0.15s ease;
+  margin-bottom: 2px;
+}
```

Comment:
IDK where it would be but I wish we could put this style somewhere other than the shame file